Dockerfile.assets

```diff
@@ -8,16 +8,14 @@ USER root
 COPY .docker/parameters.yml app/config/
 
 RUN mkdir web && \
-    bin/console assets:install
+    SYMFONY__API_URL=https://api.elifesciences.org bin/console assets:install
```

Comment:
Bit annoying. We should switch to using proper environment variables which would allow setting sensible defaults, but involves formula changes so let's wait.

Reply:
Not sure why `assets:install` needs this, does it load remote content?

Reply:
No, Symfony just needs the parameter to be set at the moment.

Reply:
As in the nginx configuration could it be a local value?